biome.jsonc

```diff
@@ -167,69 +147,16 @@
         }
       },
       "suspicious": {
-        // `void` as type is useful in our case when used as generic constraint e.g. K extends number | void
-        "noConfusingVoidType": "off",
         // There is a lot of empty code blocks, should be enabled and clean up separately.
         "noEmptyBlockStatements": "off",
         "noConsoleLog": "error"
```

Comment:
upstream?

Reply:
On upstream we have it as `warn` level. Not sure if we enforce it as `error` from upstream for all projects. What do you think?